compiler/test/unit/lexer/location_offset.d

```diff
@@ -16,7 +17,7 @@ unittest
 {
     enum code = "token";
 
-    scope lexer = new Lexer("test.d", code.ptr, 0, code.length, 0, 0);
+    scope lexer = new Lexer("test.d", code.ptr, 0, code.length, 0, 0, new ErrorSinkStderr);
```

Comment:
I think this should just be a function at this point, instead of copy-pasted code

Reply:
It's a class because it consists of several functions, not one.

Reply:
I meant that at some point, we should realize that:
```D
scope lexer = new TestLexer(code);
// Or:
scope lexer = makeTestLexer(code);
```

is the way to go.

Reply:
That's worth considering for a future PR. We already use both techniques in dmd.